modules/videosipgw/JitsiVideoSIPGWSession.ts

```diff
@@ -129,8 +134,13 @@ export default class JitsiVideoSIPGWSession extends Listenable {
      * @private
      * @param {string} action - The action to send ('start' or 'stop').
      */
-    _sendJibriIQ(action) {
-        const attributes = {
+    private _sendJibriIQ(action: string): void {
+        const attributes: {
+            xmlns: string;
+            action: string;
+            sipaddress: string;
+            displayname?: string;
```

Comment:
why not add the displayname directly here? that way you won't need to define the shape of the object.